docs/css/customstyles.css

```diff
@@ -1102,11 +1102,6 @@ h4.panel-title {
   text-align: center;
 }
 
-.modal-demo-bg{
-    background-color:rgba(0, 0, 0, 0.5);
-    padding: 40px;
-}
-
```

Comment:
@InnaAtanasova do you think we can just remove this file entirely and cleanup `head_print.html`? Or maybe even delete `head_print`? It's referencing a ton of css that doesn't exist anymore and I'm not sure we still need it.